addons/udes_stock/models/stock_inventory.py

```diff
@@ -38,6 +49,14 @@ class StockInventory(models.Model):
         "Start Date Scanning",
         help="The date operator started scanning."
     )
+    user_id = fields.Many2one(
+        "res.users",
+        string="Checked Owner",
+        readonly=True,
+        states={"draft": [("readonly", False)], "waiting": [("readonly", False)]},
+        help="Specify Owner to focus your stock check on a particular user.",
+        domain=_get_stock_check_users,
```

Comment:
Q: Does this method get called each time we check for a user, or only when the Odoo process starts?

For example if Bob isn't a stock user, but then I make him one through the UI, will he appear in this dropdown without needing to restart Odoo?

Probably fine just something I want to double check because I don't use a method on domain that often. :)